.vscodeignore

```diff
@@ -1,28 +1,22 @@
-.github/**
-.vscode/**
-.vscode-test/**
-.changes/**
-scripts/**
-buildspec/**
-dist/src/test/**
-dist/src/integrationTest/**
-dist/src/testFixtures/**
-dist/test/**
-dist/integrationTest/**
-dist/third-party/test/**
-dist/**/*.map
-src/**
-out/**
-third-party/test/**
-webpack.config.js
-.gitignore
-.nycrc.json
-Dockerfile
-tsconfig.json
-tslint.json
-codebuild-image-buildspec.yml
-node_modules
-docs/**
-designs/**
-coverage/**
-package.json.bk
+# Ignore everything by default. #1899
+*
+*/**
+
+# Allowlist
+!dist/*
+!dist/libs
+!dist/*/!(testFixtures|test)/**/!(*.*.map)
+!dist/*/!(*.*.map)
+!media/**
+!resources/**
+!syntaxes/**
+!templates/**
+!third-party/**
```

Comment:
I would expect `dist/` to contain all of the things actually wanted for distribution. I guess that's another flaw in our current build scripts?